dora/underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -395,6 +395,10 @@ public Boolean hasNextChunk() {
   @Override
   protected ObjectStatus getObjectStatus(String key) {
     try {
+      if (isRoot(key)) {
+        // return a virtual root object
+        return new ObjectStatus(key, null, 0, null);
```

Comment:
what are the attributes from the user side?

Reply:
user can not see the root directly attribute